app/services/add_users.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class AddUsers 
```

Comment:
Here and a few other spots there is whitespace that the `rubocop` linter does not like. This is causing the build to fail. (I should mention rubocop in the PR template I just added.)

Depending on your code editor, you can probably install a rubocop plugin to see such things while you're editing. You can also run `rubocop` from the command line to see a list of linter violations.